api/pyproject.toml

```diff
@@ -4,16 +4,16 @@ build-backend = "poetry.masonry.api"
 
 [tool.poetry]
 name = "doctr-api"
-version = "0.5.2a0"
+version = "1.0.1a0"
 description = "Backend template for your OCR API with docTR"
 authors = ["Mindee <contact@mindee.com>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.11"
+python = ">=3.8.2,<3.11"
 tensorflow = ">=2.9.0,<3.0.0"
 tensorflow-addons = ">=0.17.1"
-python-doctr = ">=0.2.0"
+python-doctr = { version = ">=1.0.0", extras = ['tf'] }
```

Comment:
Correct me if I'm wrong, but I think this will result as double version specifiers. If we consider extras, we need to remove the "tensorflow" & cie specifiers

Reply:
i don't think i understand your comment can you be more clear? 
we added the extras because now doctr if you don't install the extras "tf" you don't have tf2onnx library and things like that, that are needed to run the code.

Reply:
Yes sure, I wasn't referring to the extra but the version index specifier
You switched it from `>=0.2.0` to `>=1.0.0` which is a big deal :sweat_smile:

Reply:
Regarding the extra, I'd argue it's up for discussion but the API doesn't need to convert models to ONNX

Ideally, we'd like it to run on ONNX, but some postprocessing ops are not compatible yet. So the alternative is to use doctr, and then carefully add as little deps as possible to run it

Reply:
I agree. but for now how doctr is done to run the code in the api tf2onnx is needed